revamp-service/src/main/java/com/revamp/core/controller/SchoolController.java

```diff
@@ -88,19 +88,29 @@
 		return ResponseEntity.ok().body(list);
 	}
 
+	@PutMapping("/updateSchool/{id}/{status}")
+	public ResponseEntity<School> updateUser(@PathVariable long id, @PathVariable String status) {
+		System.out.println("Update school"+id+""+status);
```

Comment:
System.out.println can be replaced with logger.info.
For eg: logger.info("Update school {} {}", id, status);